### ISSUE TITLE
Issue running install in Visual Studio 2019

### ISSUE THREAD

**Issue by BramvdBogaard** [jakobehn/GitFlow.VS#104]
I'm having an issue with installing GitFlow in visual studio 2019.
When going to the GitFlow tab in Team Explorer I see the following message (even though I've run gitflow init in cmd)
![image](https://user-images.githubusercontent.com/33619813/146745058-d60e21b8-b4e8-46c5-bce2-6d2e93e99256.png)

When I press the "install" button an error message very briefly appears which reads as follows:

`VERBOSE: Performing the operation Copy File on target C:\users\<user>\appdata\local\microsoft\visualstudio\16.0.8_855f8d03\extensions\ontyaeup.31v\Dependencies\binaries\getopt.exe

Copy-item: Can't find part of path C:\users\<user>\appdata\local\microsoft\visualstudio\16.0.8_855f8d03\extensions\ontyaeup.31v\Dependencies\binaries\getopt.exe

At C:\users\<user>\appdata\local\microsoft\visualstudio\16.0.8_855f8d03\extensions\ontyaeup.31v\Dependencies\install.psl:19 char:1
Copy-Item -Path "$binaries\*.*" -Destination "$gitLocation" -Force -V ...`

I have no clue how to fix this issue and as a result of this I cannot use GitFlow currently. Any help would be appreciated!

**Comment by jakobehn** [comment]
Hmm, can't reproduce this error. I just published a new version for VS2022, could you perhaps try this one and see if you the the same error?

https://marketplace.visualstudio.com/items?itemName=vs-publisher-57624.GitFlowforVisualStudio2022

**Comment by BramvdBogaard** [comment]
Hi Jakob,
Glad to see that the 2022 version is out!

Unfortunately I get the exact same error as I did in 2019.
The following error translates to "Can't find part of path"

![image](https://user-images.githubusercontent.com/33619813/148189853-d4feb64f-0b3f-4a64-9fda-a26201809297.png)

**Comment by jakobehn** [comment]
That's strange. Can you check what has been installed into your extensions folder? Mine looks like this after the installation:

![image](https://user-images.githubusercontent.com/847244/148205689-11515e77-980c-4b9e-83f2-76f90e97397f.png)

**Comment by BramvdBogaard** [comment]
Thanks for your quick reply!
This is what my folder looks like, I have only 1 folder under "extensions" and that is this one.

![image](https://user-images.githubusercontent.com/33619813/148448427-e27eb046-8483-4913-8d7e-b27e10d9974d.png)

**Comment by jakobehn** [comment]
Thanks. The error message says that it cant locate the **getopt.exe** file that should be located in the **Dependencies\binaries** folder. Can you verify that the file is there?

**Comment by BramvdBogaard** [comment]
Hi,
I do have the getopt.exe file there.
![image](https://user-images.githubusercontent.com/33619813/148681715-fd0414ae-cdfb-47a9-a4c0-9eb1488d7bbb.png)

**Comment by BramvdBogaard** [comment]
It seems we are looking for the wrong file. The error points towards `Dependencies/Git/cmd/bin` . This file does indeed not exist because the Git-folder doesn't exist under Dependencies. 

![image](https://user-images.githubusercontent.com/33619813/148771578-3a9d31ac-2c8b-4e2a-a47a-7585150bc303.png)

**Comment by jakobehn** [comment]
Ah OK, I think that the code can't find the Git installation path. Have you installed Git for Windows? It's listed as a pre-requirement for this extension

https://gitforwindows.org/

**Comment by BramvdBogaard** [comment]
Yes I do have Git for Windows installed. Yet the Git folder doesn't show up.
I have compared the files with the files on a system that has GitFlow working. That system also does not have the Git folder so that means that my install is probably correct.

Could there be an error in the script determining where to look for the Git installation?

**Comment by jakobehn** [comment]
Maybe, here is the code that gets the Git installation directory:
https://github.com/jakobehn/GitFlow.VS/blob/26561e23aaacd619984cc00ce13784e0ed124a50/GitFlow.VS/GitHelper.cs#L24
